omod/src/main/java/org/openmrs/module/patientflags/web/FindFlaggedPatientsController.java

```diff
@@ -159,9 +159,21 @@ public ModelAndView processSubmit(@ModelAttribute("filter") Filter filter, Bindi
 		
 		// create the model map
 		ModelMap model = new ModelMap();
-		model.addAttribute("flaggedPatients", flaggedPatients.getMemberIds());
 		model.addAttribute("allPatients", allPatients);
-		
+		List<Map<String, Object>> flaggedPatientList = new ArrayList<>();

```

Comment:
Can't we use Map<String, Integer> here?

Reply:
yes we can. I updated map value type object to Integer.